package.json

```
@@ -20,12 +20,14 @@
     "@types/react-dom": "^17.0.0",
     "@typescript-eslint/eslint-plugin": "^4.5.0",
     "@typescript-eslint/parser": "^4.5.0",
+    "axios": "^0.23.0",
```

Can we use a specific version for this dependency?
```suggestion
    "axios": "0.23.0",
```

---

Resolved in the rebase :)